src/helpers/hooks.ts

```diff
@@ -0,0 +1,12 @@
+import { useEffect, useState } from 'react';
+
+/**
+ * Custom hook to determine if the component is mounted.
+ */
+export const useIsMounted = () => {
```

Comment:
We already have `useIsClient` for this. If you find time please reuse it. Or we can keep both for some time, it doesn't hurt anything. 🙂 

One sidenote: please do not add comments in code. I know it looks very tidy, but the problem with comments is, that after a while the code will be updated, but comment will stay obsolete, thus wrong. It is always better to make the code "self documenting" and that is achieved by proper naming. That is also the reason, why I am fairly strict on namings.

There are few exceptions to add comments -eg: hard algorithms, cumbersome code solving a bug, or TODOs.